tempesta_fw/http_parser.c

```diff
@@ -1455,10 +1453,16 @@ tfw_http_parse_req(void *req_data, unsigned char *data, size_t len)
 
 	__FSM_START(parser->state) {
 
+	/* ----------------    EOL   ---------------- */
+
+	#define TFW_HTTP_REQ_EOL
+	#include "http_parser_eol.h"
+	#undef TFW_HTTP_REQ_EOL
```

Comment:
The parser code is organized to minimize size of jumps as much as possible, e.g. you probably noted that "improbable" states are placed at the end of parsing functions. So it's better to place EOL processing code somewhere near to finishing header lines or starting a new header.

Yes, the parser code is large and clumsy, but moving only one state to other file doesn't make the code better. I though about reworking the logic to 3 files: common definitions and functions and request and response processing. I think it'll be good to do at the time of reworking current code to use SSE.